frontend/landing.html

```diff
@@ -0,0 +1,652 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Art Trend Classifier - AI Art Analysis</title>
+    <style>
+        * {
```

Comment:
This CSS is a piece of shit. It is not readable, not maintanable. You should use something like https://tailwindcss.com/.

Honestly this code would not enter any serious project.